uint64 timestamp		# Timestamp in microseconds since boot

uint8[51] text
uint8 severity
